### PR TITLE
Add advice for inefficient satyr combos

### PR DESCRIPTION
**Pull request by jumboduck**

### CODE REVIEW

File: src/helpers/getDeckAdvice/advice/INEFFICIENT_SATYR_COMBOS.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import toSentence from '../../toSentence/'
+import getRawCardData from '../../getRawCardData/'
+
+const getSatyrs = cards => cards.filter(c => c.race === 'satyr')
```
Comment (KittySparkles) [inline src/helpers/getDeckAdvice/advice/INEFFICIENT_SATYR_COMBOS.js:4]:
```suggestion
const getSatyrs = cards => cards.filter(card => card.race === 'satyr')
```

File: src/helpers/getDeckAdvice/advice/INEFFICIENT_SATYR_COMBOS.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+const hasAny = (cards, ids) =>
+  ids.some(id => cards.map(card => card.id).includes(id))
+
+export default cards => {
+  const hasSatyrSynergist = hasAny(cards, ['S5', 'S7', 'S9'])
+  const satyrs = getSatyrs(cards)
+
+  if (!hasSatyrSynergist || satyrs.length > 5) return null
```
Comment (KittySparkles) [inline src/helpers/getDeckAdvice/advice/INEFFICIENT_SATYR_COMBOS.js:12]: Is 5 a fine number?

File: src/helpers/getDeckAdvice/advice/INEFFICIENT_SATYR_COMBOS.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+  const hasCard = id => cards.map(card => card.id).includes(id)
```
Comment (KittySparkles) [inline src/helpers/getDeckAdvice/advice/INEFFICIENT_SATYR_COMBOS.js:14]: We can rewrite `hasAny` like this:
```js
  const hasCard = id => cards.map(card => card.id).includes(id)
  const hasAny = ids => ids.some(hasCard)
```

File: src/helpers/getDeckAdvice/advice/INEFFICIENT_SATYR_COMBOS.js [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+    'and'
+  )
+
+  return {
+    id: 'INEFFICIENT_SATYR_COMBOS',
+    name: 'Inefficient Satyr Combos',
+    description:
+      'This deck includes ' +
+      listOfNames +
+      ', but doesn’t include enough satyrs to provide good synergy. Consider including more satyrs.',
```
Comment (KittySparkles) [inline src/helpers/getDeckAdvice/advice/INEFFICIENT_SATYR_COMBOS.js:30]: We could use a template literal here to make it a little nicer to read. :)

File: src/helpers/toSentence/spec.js [hunk 5]
```diff
@@ -0,0 +1,9 @@
+import toSentence from './'
+
+describe('The `toSentence` helper', () => {
+  it('should turn an array into a sentence', () => {
```
Comment (KittySparkles) [inline src/helpers/toSentence/spec.js:4]: Can we add a test for 0 entry, and 1 entry as well? :)

File: src/helpers/getDeckAdvice/advice/INEFFICIENT_SATYR_COMBOS.js (hunk 2 above)
Comment (jumboduck) [reply src/helpers/getDeckAdvice/advice/INEFFICIENT_SATYR_COMBOS.js:12]: I seldom use this faction, so I'm not sure. Might need input from a swarm veteran.

**Pull request merged**